db/migrate/20240722151857_create_join_table_customers_items.rb

```diff
@@ -0,0 +1,8 @@
+class CreateJoinTableCustomersItems < ActiveRecord::Migration[7.1]
+  def change
+    create_join_table :customers, :items do |t|
+      t.index [:customer_id, :item_id]
+      t.index [:item_id, :customer_id]
+    end
```

Comment:
I'm curious how you created this migration. Remember you have Rails generators and could type something like: `rails g migration CreateCustomerItems customer:references item:references` and that would create this migration file for you with the foreign keys set up already instead of this bigint and index. This migration _works_ though, just to be clear.